libchannelserver/ChannelSession.h

```diff
@@ -155,7 +163,8 @@ class ChannelSession : public std::enable_shared_from_this<ChannelSession>
 
     std::map<std::string, ResponseCallback::Ptr> _responseCallbacks;
 
-    std::shared_ptr<std::set<std::string> > _topics;
+    mutable dev::SharedMutex x_topics;
+    std::set<std::string> _topics;
```

Comment:
Plz follow the code standard, make the member begin with m_

Reply:
done, pls refer to: https://github.com/FISCO-BCOS/FISCO-BCOS/pull/490/files#diff-0cbd99faad88554b405f24eb9d6cf943